README.rst

```diff
@@ -41,8 +41,30 @@ Control machine requirements:
 
 - Ansible 2.1+ installed. See `Ansible install docs`_
 
+Initial set-up
+--------------
+First, clone the jupyterhub-deploy-teaching repository to your machine and navigate to the folder.
+
+.. code-block:: 
+
+   git clone https://github.com/LibreTexts/jupyterhub-deploy-teaching
```

Comment:
Change this to mechmotum instead of LibreTexts as it is our "primary" repo.

Reply:
Fixed.